### PR TITLE
Add information about View mode to Design Editor

### LINKED ISSUE #279: Document Read-only viewer behavior for Git projects
From Will:

Billie Hilton (aka will) 
Currently, opening Studio in read-only mode for Stoplight “Viewer” roles is impossible for git projects, because Studio runs git clone with your user’s VCS account. So you must connect your VCS account first. Hence you are seeing the “Connect to Git” button.
However as part of USP, once a project is connected to a Service Account, and we can use the Service Account to do the git clone on the server instead of in the browser, then we can provide Viewer roles with a read-only Studio mode (edited) 

Slack discussion:
https://stoplight-internal.slack.com/archives/CELPZGMA4/p1646337411306129

**This is current behavior that we do not have documented but probably should:**

From Marc:
it is for the purposes that troy and andrew are describing - it is not the same as docs view - a lot of folks that I talked with prefer to “review” things in the context of studio. and we have a share button inside of studio - granting somebody read access and then sharing the current URL is a natural thing to do

### PR DESCRIPTION
**Pull request by heitortsergent**
Closes #279.

### CODE REVIEW

File: 3.-design/api-design-editor.md [hunk 1]
```diff
@@ -1,20 +1,38 @@
 # Design Editor Overview
 
-Use the Design Editor, also referred to as Stoplight Studio, to design and manage APIs, documentation, and related files stored in projects. Projects work and look the same in Studio Web and Studio Desktop, with a few differences. Learn more about the differences in our [design overview](design-overview.md##studio-web-vs-studio-desktop).
+Use the Design Editor, also referred to as Stoplight Studio, to view, design, and manage APIs, documentation, and related files stored in projects.
+
+Workspace Guests can't access the Design Editor.
```
Comment (pamgoodrich) [inline 3.-design/api-design-editor.md:5]: I think "Guests" should be "guests." Can you see how we're using this in other places?

File: 3.-design/api-design-editor.md [hunk 2]
```diff
@@ -1,20 +1,38 @@
...
+
+### Edit Project
+
+Project owners, admins, and editors can select **Edit** to manage the project.
+
+![Highlighting the Edit button on the Project sidebar of a project, viewed as an Project Editor or higher role](../assets/images/stoplight-project-editor-edit.png)
+
+### View Project
+
+For web projects, and Git projects that have a [service account](../2.-workspaces/configure-git/h.service-accounts.md) configured, project viewers can select **View** to view the project.
```
Comment (pamgoodrich) [inline 3.-design/api-design-editor.md:26]: No need for the commas to separate the Git clause.

File: 3.-design/api-design-editor.md [hunk 3]
```diff
@@ -41,45 +59,52 @@ You can customize the published [project sidebar](https://meta.stoplight.io/docs
...
-  - **Discard**: Once changes are made to a project, select the down arrow to discard them. 
-  > For web projects, discarding impacts unpublished changes made by all users and not just the user performing the discard.
 
-**Project Name**: Shows the name of the current project. The version is shown for Stoplight projects. For projects connected to a repository, the name of the current branch is shown. Select the down arrow to pull changes into your project or to switch branches. 
+- **Publish**: When the project isn't connected to a repository, this option is enabled when at least one change has been made in the project. 
+- **Commit and Publish**: When the project is connected to a repository, this option is enabled when at least one change has been made in the project.  
+- **Discard changes since last publish**: Once changes are made to a project, select the down arrow next to **Publish** or **Commit and Publish** to see this option, and select it to discard any changes. 
+
+For Viewers, the **Project Action** section is greyed out at the top and users can't make changes or add new files.
```
Comment (pamgoodrich) [inline 3.-design/api-design-editor.md:90]: greyed out > disabled

**Review by pamgoodrich — APPROVED** [review]
A few minor suggestions. Nice job overall!

File: 3.-design/api-design-editor.md (hunk 1 above)
Comment (heitortsergent) [reply 3.-design/api-design-editor.md:5]: You're right, we use lowercase in other places. Fixed!

**Comment by heitortsergent** [comment]
@pamgoodrich thanks for the review!

**Pull request merged**